tests/clipboard/clipboard_plugin.test.ts

```diff
@@ -2001,6 +2005,127 @@ describe("clipboard", () => {
     expect(getCell(model, "C3")!.content).toBe(expectedInvalidFormula);
   });
 
+  test("filtered rows are ignored when copying range", () => {
+    const model = createModelFromGrid({
+      B2: "b2",
+      C2: "c2",
+      D2: "d2",
+      B3: "b3",
+      C3: "c3",
+      D3: "d3",
+      B4: "b4",
+      C4: "c4",
+      D4: "d4",
+    });
+
+    createTableWithFilter(model, "B2:D4");
+    updateFilter(model, "C3", ["c3"]);
+    copy(model, "B2:D4");
+    paste(model, "B5");
+
+    expect(getEvaluatedGrid(model, "B5:D7")).toEqual([
+      ["b2", "c2", "d2"],
+      ["b4", "c4", "d4"],
+      ["", "", ""],
+    ]);
+  });
+
+  test("hidden row are taken into account when copypasting range", () => {
+    const model = createModelFromGrid({
+      B2: "b2",
+      C2: "c2",
+      D2: "d2",
+      B3: "b3",
+      C3: "c3",
+      D3: "d3",
+      B4: "b4",
+      C4: "c4",
+      D4: "d4",
+    });
+
+    hideRows(model, [2]);
+    copy(model, "B2:D4");
+    paste(model, "E1");
+
+    expect(getEvaluatedGrid(model, "E1:G3")).toEqual([
+      ["b2", "c2", "d2"],
+      ["b3", "c3", "d3"],
+      ["b4", "c4", "d4"],
+    ]);
+  });
+
+  test("hidden row are taken into account when cutpasting range", () => {
```

Comment:
```suggestion
  test("hidden rows are taken into account when cutpasting range", () => {
```

Reply:
should also test that filted rows are kept on cut/paste

nitpick: could also test both hidden rows/cols in a single test if we want to gain some lines of code